src/Moq/Properties/Resources.resx

```diff
@@ -346,6 +346,10 @@ Expected invocation on the mock once, but was {4} times: {1}</value>
 	</data>
 	<data name="UnverifiedInvocations" xml:space="preserve">
 		<value>The following invocations were not verified:
+{0}</value>
+	</data>
+	<data name="MultipleVerificationErrors" xml:space="preserve">
```

Comment:
Please remove this, along with the change in `Resources.Designer.cs`.

Instead, as mentioned in another review comment, change the value of `UnverifiedInvocations` (right above these lines) to include a placeholder for the mock's name. (See other review comment for details.)